openqaoa/problems/converters.py

```diff
@@ -18,7 +18,7 @@
 
 
 class FromDocplex2IsingModel:
-    def __init__(self, model, multipliers: float = None):
+    def __init__(self, model, multipliers: [float, list] = None):
```

Comment:
For the multipliers here. Are the types float or list, or float and list?? If float or list, we should be using Union[float, list] here.

Reply:
Thanks @shahidee44, I updated it!